### PR TITLE
Updated interaction model and manifest, renamed HelloWorld intent to QuestionIntent

### PR DESCRIPTION
**Pull request by paulotruta**
Also did a tidy up to comments and code lines that were wrongly formatted.

### CODE REVIEW

File: skill.json [hunk 1]
```diff
@@ -26,23 +26,44 @@
       }
     },
     "manifestVersion": "1.0",
+    "privacyAndCompliance": {
+      "locales": {
+        "en-US": {
```
Comment (inverse) [inline skill.json:31]: Should these point to the repo?

File: skill.json [hunk 2]
```diff
@@ -26,23 +26,44 @@
...
+      "automaticDistribution": {
+        "isActive": false
+      },
+      "category": "EDUCATION_AND_REFERENCE",
       "distributionCountries": [],
       "isAvailableWorldwide": true,
       "locales": {
         "en-US": {
-          "description": "Inteface with chatGPT using voice! The openAI powered chatbot will respond to your queries and even maintain a conversation with you.",
+          "description": "Have the power of chatGTP at the tip of your tonge! Alexa can relay questions to chatGPT and provide you the answers in natural language! This is an excellent way to get to know about certain topics in detail, or brainstorm a fantasy world!",
```
Comment (inverse) [inline skill.json:50]:
```suggestion
          "description": "Have the power of chatGPT at the tip of your tongue ! Alexa can relay questions to chatGPT and provide you the answers in natural language! This is an excellent way to get to know about certain topics in detail, or brainstorm a fantasy world!",
```

File: skill.json [hunk 3]
```diff
@@ -26,23 +26,44 @@
...
           "examplePhrases": [
             "Alexa ask jee pee tee a question",
-            "Alexa, ask jee pee tee about our solar system",
-            "help"
+            "ask gee pee tee how to boil an egg",
+            "ask gee pee tee to explain in detail about the NP theorem"
+          ],
+          "keywords": [
+            "gpt",
+            "chatGPT",
```
Comment (inverse) [inline skill.json:58]:
```suggestion
            "ChatGPT",
```

File: skill.json [hunk 4]
```diff
@@ -26,23 +26,44 @@
...
+            "AI"
           ],
-          "keywords": [],
-          "name": "Gee Pee Tee (chatGPT)",
-          "summary": "A relay to chatGPT, the openAI powered chatbot."
+          "name": "Gee Pee Tee (Chat GTP)",
+          "summary": "A skill that lets you ask questions to chat GTP!",
```
Comment (inverse) [inline skill.json:62]:
```suggestion
          "summary": "A skill that lets you ask questions to ChatGPT!",
```

**Review by inverse — DISMISSED** [review]
Looks good aside minor details :+1:

**Review by inverse — APPROVED** [review]

**Comment by paulotruta** [comment]
Thank you for the review @inverse! I additionally added a privacy policy and terms of use files that we can use to link in the skill manifest 👍

Will leave this PR open for some time in case some more detail comes up.

**Review by inverse — APPROVED** [review]
Looks good :)

**Pull request merged**